2-Draft-Accepted/RFCxxxx-Secrets-Management.md

```diff
@@ -45,8 +46,8 @@ Registering and using remote secrets:
 Install-Module Az.KeyVault
 
 # In this example, we explicitly register this extension
-Register-SecretsVaultExtension -Name Azure -Cmdlet Get-AzKeyVaultSecret `
-  -Module Az.KeyVault -Version 1.0.0
+Register-SecretsVault -Name Azure -Cmdlet Get-AzKeyVaultSecret `
```

Comment:
Should be Register-SecretVault (pluralization in cmdlet names)

Reply:
Not true. Seems like you missed [@SteveL-MSFT's reply about that](https://github.com/PowerShell/PowerShell-RFC/pull/208#discussion_r302683489). `SecretsVault` is singular.

Reply:
While there might only be one "SecretsVault", and it might actually contain multiple "Secrets", the point of the "No Plurals" rule is about how plurals translate.  At this point in the ecosystem, it's also about a reasonable expectation.

I do not believe that any PowerShell user will assume that Register-SecretVault will register a vault and then make it secret (it kinda flies in the face of the cmdlet name).  I do believe that, especially as we have "Add-Secret", "Register-SecretsVault" would end up being confusing in practice.  Your brain would remember one or the other, and you'd be more likely to flub it.

The other rationale, past pure pluralization, is the desire for common noun roots and the easy memorization that comes with it.

Reply:
`SecretsVault` indicates a **single** vault containing multiple secrets. `SecretVault` indicates a single vault that itself is secret. I am with @KirkMunro on this one, the proposed noun is singular as the noun itself here is `Vault` while `Secrets` is describing what the vault itself is for.